flask_codemirror/__init__.py

```diff
@@ -50,7 +51,9 @@ class CodeMirrorHeaders(object):
     THEME_KEY = 'CODEMIRROR_THEME'
     ADDONS_KEY = 'CODEMIRROR_ADDONS'
     VERSION_KEY = 'CODEMIRROR_VERSION'
+    SERVE_LOCAL_KEY = 'CODEMIRROR_SERVE_LOCAL'
     CDN_URL = '//cdnjs.cloudflare.com/ajax/libs/codemirror/{0}/'
+    LOCAL_PATH = '/codemirror/{0}'
```

Comment:
I think it should be better if there was a config key for this path, since it's local users could host the scripts where they want.